src/c++/perf_analyzer/docs/examples/profile.py

```diff
@@ -307,7 +416,7 @@ def construct_input_data(args):
     parameters set by input JSON file.
     """
     prompt = ""
-    stream = True
+    stream = False
```

Comment:
Why change the default? This implies when people run `profile.py` without specifying `--streaming`, they're doing an offline analysis?

Reply:
Is the default being switched simply because of the fact that the easiest-to-understand command line option has streaming in the affirmative? i.e. it'd be awkward to have `--no-streaming`?

Reply:
I think we should make the default the more useful case, regardless of if the option name because more awkward.

Reply:
Changed to `args.offline` and made online as default.